spec/requests/like_spec.rb

```diff
@@ -2,9 +2,8 @@
 
 RSpec.describe 'Likes', type: :request do
   describe 'GET /create' do
-    it 'returns http success' do
-      get '/like/create'
-      expect(response).to have_http_status(:success)
+    describe 'GET /index' do
+      pending "add some examples (or delete) #{__FILE__}"
     end
```

Comment:
- [ ] Let's implement all the pending tests, and delete those you think are not necessary.